src/ws/cockpitauth.c

```diff
@@ -821,6 +832,9 @@ cockpit_auth_spawn_login_async (CockpitAuth *self,
 
   command = type_option (type, "command", cockpit_ws_session_program);
 
+  /* Get the wanted authfd for this command, default is 3 */
+  child_fds[1] = type_guint_option ("authFD", type, 3, 1024, 3);
```

Comment:
I wish there was a more clear name for this. I guess it's a rarely used option ... so maybe not a big deal.